rdsn/thirdparty/CMakeLists.txt

```diff
@@ -156,6 +156,33 @@ ExternalProject_Add(thrift
         DEPENDS boost
         )
 
+# kerberos
+ExternalProject_Add(krb5
```

Comment:
Can't the two libs be installed directly on centos6?

Reply:
Why not add centos6 Dockerfile if you insist to support centos6?

Reply:
centos6 has stopped supporting and I haven't find the open software yum-repo.  I have build centos6-docker-env in our company internal

Reply:
> Can't the two libs be installed directly on centos6?

the version is too old to be compatibility with `zookeeper`

Reply:
Have you tried https://www.getpagespeed.com/server-setup/how-to-fix-yum-after-centos-6-went-eol? I've test that the two libs can be installed.
But I think there must be some other problems to build on centos6, it's better to create a centos6 Dockerfile to resolve all the problems.
And, for the non-centos6 platforms, we'd better skip build the two packages since we have installed the packages.

Reply:
I think if one package has compatibility problem in different env, the best solution is specify unified version(if we can find the version) in thirdparty, but not use different version in different env.

So I think the two package actually should be included in thirdparty though in centos7, unless one day we no longer support centos6

Reply:
> Have you tried https://www.getpagespeed.com/server-setup/how-to-fix-yum-after-centos-6-went-eol? I've tested that the two libs can be installed.
> But I think there must be some other problems to build on centos6, it's better to create a centos6 Dockerfile to resolve all the problems.
> And, for the non-centos6 platforms, we'd better skip build the two packages since we have installed the packages.

Great, I will try it

Reply:
> I think if one package has compatibility problem in different env, the best solution is specify unified version(if we can find the version) in thirdparty, but not use different version in different env.

Agree. But only limit to the platforms we plan to support, centos 6 is EOL on 2020-12, I don't think it's worthy to support it.

Reply:
Yeah, I actually don't want support the old version, but our company need it, and maybe need some time to upgrade the centos7.

Reply:
> > Have you tried https://www.getpagespeed.com/server-setup/how-to-fix-yum-after-centos-6-went-eol? I've tested that the two libs can be installed.
> > But I think there must be some other problems to build on centos6, it's better to create a centos6 Dockerfile to resolve all the problems.
> 
> Great, I will try it

I have test and the repo can be used